setup.cfg

```
@@ -125,6 +125,7 @@ filterwarnings =
     ignore:::astropy.tests.disable_internet
     ignore:direct construction of Asdf:pytest.PytestDeprecationWarning
     ignore:Unknown pytest.mark.mpl_image_compare:pytest.PytestUnknownMarkWarning
+    ignore:Unknown config ini key:pytest.PytestConfigWarning
```

```suggestion
    ignore:Unknown config ini key:pytest.PytestConfigWarning
    ignore:Unknown config option:pytest.PytestConfigWarning
```
Seems we need both to get rid of all warnings, which otherwise is now with 6.1 causing a failure early on (due to our `xfail_strict = true`?).
So this would actually be needed quite urgently to fix #10773 .

---

Thanks for the comment. I see that they released pytest yesterday during a weekend. I'll investigate on Monday.

---

Yes, looks like all tests are failing right now. So you might give it a try with the `ignore: Unknown config option` added to see if that passes, or if we also need the
`ignore:The imp module is deprecated in favour of importlib:DeprecationWarning`

---

It seems they renamed the warning so the "Unknown config option" line would be sufficient.
@dhomeier - all warnings are turned into error with :  
```
filterwarnings =
    error
    ...
```

---

```suggestion
    ignore:Unknown config option:pytest.PytestConfigWarning
```

---

I committed the change for the warning to see if it fixes the CIs.